### PR TITLE
chore: python demo

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This is a demo related to #43836 - we should review this heavily to use as a reference point for migrating the other projects.

### CODE REVIEW

**Comment by ieahleen** [comment]
I think it works

**Review by majestic-owl448 — DISMISSED** [review]
With Oliver fix even better

LGTM :+1:

**Review by ShaunSHamilton — COMMENTED** [review]
I think I am happy with this. I assume we need to create a mass issue to keep track of these, and remove the instructions from the boilerplates?

Personally, I would still copy this over to Replit, because it is annoying to have the instructions on a different platform.

**Review by ejcheng — DISMISSED** [review]

**Review by gikf — DISMISSED** [review]
As a mass issue can be used #43836, just a matter of opening it for contributions. And possibly pinning for a little more exposure.

File: curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/arithmetic-formatter.md [hunk 1]
```diff
@@ -8,22 +8,87 @@ dashedName: arithmetic-formatter
...
+
+```py
+  235
++  52
+-----
+```
+
+Create a function that receives a list of strings that are arithmetic problems and returns the problems arranged vertically and side-by-side. The function should optionally take a second argument. When the second argument is set to `True`, the answers should be displayed.
+
+## For example
```
[the remark this answers is not in the document]
Comment (ieahleen) [reply curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/arithmetic-formatter.md:30]: Last thing left to do seems to decide if it should be an "example" or a "for example"

**Review by majestic-owl448 — APPROVED** [review]

**Review by ShaunSHamilton — APPROVED** [review]

**Pull request merged**